components/Skeleton/Skeleton.unit.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Skeleton from './Skeleton';
+
+describe('Skeleton component', () => {
+  it('Should match the snapshot of a simple skeleton', () => {
+    const simpleSkeleton = <Skeleton />;
+    expect(renderer.create(simpleSkeleton).toJSON()).toMatchSnapshot();
+  });
+
+  it('Should match the snapshot of a text skeleton', () => {
+    const simpleSkeleton = <Skeleton.Text />;
```

Comment:
nitpicking: here the vars are all `simpleSkeleton` but only one of them is really `simple`. You can change all of them to `skeleton` ou customize one by one with their respective name like `circleSkeleton`.